src/core/server/rendering/rendering_service.tsx

```diff
@@ -124,6 +124,10 @@ export class RenderingService {
         i18n: {
           translationsUrl: `${basePath}/translations/${i18n.getLocale()}.json`,
         },
+        theme: {
+          darkMode,
+          version: themeVersion,
+        },
```

Comment:
I chose to send the theme information as a new property of the injected metadata. 

The reasoning was:
- we're already going this with services such as `i18n`
- it's way simpler on the client-side than having the `theme` service depend on the `uiSettings` service
- it's future proof, because we don't know if the theme overrides will be stored in uiSettings or not when/if we implement them.